app/src/main/java/com/amaze/filemanager/ui/theme/AppTheme.java

```diff
@@ -59,14 +61,16 @@ public static AppTheme getTheme(int index) {
         return TIMED;
       case BLACK_INDEX:
         return BLACK;
+      case SYSTEM_INDEX:
+        return SYSTEM;
     }
   }
 
   /**
    * @return The Theme enum to provide to {@link
    *     com.afollestad.materialdialogs.MaterialDialog.Builder}
    */
-  public Theme getMaterialDialogTheme() {
+  public Theme getMaterialDialogTheme(boolean isNightMode) {
```

Comment:
The code for "isNightTheme" is always the same, who not write it here?

Reply:
It is indeed the same, but can we get the `Context` for doing that in that enum? This is why I had to pass that boolean everytime the method gets called!

`(c.getResources().getConfiguration().uiMode & Configuration.UI_MODE_NIGHT_MASK) == Configuration.UI_MODE_NIGHT_YES))`

Reply:
> It is indeed the same, but can we get the `Context` for doing that in that enum? This is why I had to pass that boolean everytime the method gets called!
> 
> `(c.getResources().getConfiguration().uiMode & Configuration.UI_MODE_NIGHT_MASK) == Configuration.UI_MODE_NIGHT_YES))`

Getting a Context generally hinges on whether keeping a strong reference is a good idea or not. In this case the strong reference shouldn't be kept longer than the scope of this method.

Reply:
✅ Done